roles/kubernetes-apps/ingress_controller/cert_manager/templates/deploy-cert-manager.yml.j2

```diff
@@ -2,27 +2,32 @@
 apiVersion: apps/v1beta1
 kind: Deployment
 metadata:
-  name: cert-manager
+  name: cert-manager-{{ cert_manager_controller_image_tag }}
```

Comment:
this will create multiple deployment, each time the version is upgraded

Reply:
In tasks/main.yml, we purge both legacy manifests assist and namespace before deploy the new version, so only single version will be deployed

Reply:
can you link?

Reply:
https://github.com/kubernetes-incubator/kubespray/blob/0800f7ed3ab8325302f2776e3f22a0ff82c0075b/roles/kubernetes-apps/ingress_controller/cert_manager/tasks/main.yml#L3

https://github.com/kubernetes-incubator/kubespray/blob/0800f7ed3ab8325302f2776e3f22a0ff82c0075b/roles/kubernetes-apps/ingress_controller/cert_manager/tasks/main.yml#L12